portfolio/src/main/webapp/index.html

```diff
@@ -30,10 +37,12 @@ <h3>Some Projects I've Worked On:</h3>
     <p>Johnson's Algorithm (Python)</p>
     <p>N-Dimensional Minesweeper (Python)</p>
 
-    <h3>Click Here To Get A Random Inspirational Quote:</h3>
+    <!-- Week 2 -->
+    <h3>JSON Test:</h3>
 
-    <button id="random-quote-button" onclick="addRandomQuote()">Random Quote</button>
+    <button id="comments" onclick="getServerMessageUsingArrowFunctions()">Click</button>
```

Comment:
btw "arrow functions" are called closures or sometimes lambdas. Not saying you have to change it, just some information